contracts/script/multisigTransactionProposals/safeSDK/modifyProverModeProposal.ts

```diff
@@ -48,6 +59,21 @@ async function main() {
   }
 }
 
+function processCommandLineArguments(): string {
+  const args = process.argv.slice(2); // Remove the first two args (node command and script name)
+  let command: string;
+  if (args.length === 0) {
+    console.log("No arguments provided.");
```

Comment:
The language here might want to specify that no "command" was found instead of "arguments"

Reply:
ah i feel you, updated here https://github.com/EspressoSystems/espresso-sequencer/commit/ac3bd820121b9d8b1f0a9d25bf2ef60f7353755a